ScoutSuite/providers/aws/facade/emr.py

```diff
@@ -0,0 +1,15 @@
+from ScoutSuite.providers.aws.facade.utils import AWSFacadeUtils
+from ScoutSuite.providers.aws.facade.basefacade import AWSBaseFacade
+from ScoutSuite.providers.utils import run_concurrently
+
+
+class EMRFacade(AWSBaseFacade):
+    async def get_clusters(self, region):
+        clusters_list = await AWSFacadeUtils.get_all_pages('emr', region, self.session, 'list_clusters', 'Clusters')
+        client = AWSFacadeUtils.get_client('emr', region, self.session)
+        clusters_descriptions = []
+        for cluster_id in [cluster['Id'] for cluster in clusters_list]:
```

Comment:
There's an API call behind each `client.describe_foobar`, so all `client.describe_foobar` calls should be done within `run_concurrently`. Thus, this loop should be done concurrently and parallelized like the following:

```suggestion
        cluster_ids = [cluster['Id'] for cluster in clusters_list]
        tasks = {
                asyncio.ensure_future(
                        run_concurrently(lambda: client.describe_cluster(ClusterId=cluster_id)['Cluster'])
                 ) for cluster_id in cluster_ids
        }
        for task in asyncio.as_completed(tasks):
                cluster = await task
                clusters_descriptions.append(cluster)
```